Cargo.toml

```diff
@@ -26,11 +26,11 @@ pg_test = ["serde_json"]
 
 [dependencies]
 bincode = "1.3.1"
-pgx = "0.2.5"
```

Comment:
There's a slightly easier way to do this generally, using the `[patch]` section:

```toml
[patch.crates-io]
pgx = { git = "https://github.com/timescale/pgx", branch = "promscale-staging" }
pgx-macros = { git = "https://github.com/timescale/pgx", branch = "promscale-staging" }
pgx-tests = { git = "https://github.com/timescale/pgx", branch = "promscale-staging" }
```

When placed in a workspace `Cargo.toml` (or in a single crate's `Cargo.toml`), you only have to specify the override in one place, and it will replace all uses of those crates in any of the workspace crates. In this case it's 6 of one, half a dozen of the other; but probably worth following the convention anyway.

Reply:
TIL. Thank you!